docs/source/conf.py

```diff
@@ -118,4 +118,4 @@
 myst_heading_anchors = 3
 
 # enable nice rendering of checkboxes for the task lists
-myst_enable_extensions = [ "tasklist"]
+myst_enable_extensions = ["colon_fence", "deflist", "tasklist"]
```

Comment:
not sure if its related to this PR description 🤔

Reply:
I add some [admonitions](https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html) and [definition list](https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#definition-lists) to the page that require `colon_fence` and `deflist` extensions.

Like this
https://github.com/apache/arrow-datafusion/blob/da033810b192f24a8fb63fb8e5d509e6469b482a/docs/source/user-guide/expressions.md?plain=1#L47-L50